src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -0,0 +1,338 @@
+﻿<Project>
+  <PropertyGroup>
+    <RequiresSigningXplatAPIs>true</RequiresSigningXplatAPIs>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <Description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</Description>
```

Comment:
Can this package be consumed from our myget fed?
Basically can we avoid shipping this to nuget.org. 

We already sign it, so security from the that perspective should be good.

Reply:
tracking that possibility in: https://github.com/nuget/client.engineering/issues/461